scripts/create_backup_pd

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+ZONE=us-central1-f
+BUCKET=jenkins-backup-image
+SHA=""
+TOOL_BUCKET=jenkins-backup-image
+
+while getopts z:b:s:t: arg; do
+  case ${arg} in
+    z) ZONE="${OPTARG}";;
+    b) BUCKET="${OPTARG}";;
+    s) SHA="${OPTARG}";;
+    t) TOOL_BUCKET="${OPTARG}";;
```

Comment:
please add an option for the disk name. Just in case jenkins_home already exist.